arci-ros/tests/test_rosrust_utils.rs

```diff
@@ -77,3 +77,41 @@ fn test_convert_system_time_to_ros_time() {
         ALLOWABLE_ERROR
     );
 }
+
+#[test]
+fn test_sub_helper() {
```

Comment:
What's the purpose of this test?
Is it a test of the `subscribe_helper` function?

Reply:
Your thinking is right.
I want to check working what I designed first.

Reply:
So, this is a test of the test code.
There might be room to discuss whether the test code needs to be tested or not.
Anyway, this file must contain tests only for `rosrust_utils.rs`.
If you want to test the `sucscribe_helper` function, you can write test code in the same file the function is written in (i.e. `msg_helper.rs`).